client/src/components/footer/footer.js

```diff
@@ -215,7 +217,10 @@ class Footer extends Component {
             ) : (
               <div>
                 <button
-                  className="btn tw-bg-labLightGray btn-xl text-uppercase next"
+                  //className="btn tw-bg-labLightGray btn-xl text-uppercase next"
+                  // className="btn tw-bg-labLightGray btn-xl text-uppercase next tw-border-t-labBlue tw-border-r-labBlue tw-border-8"
+
```

Comment:
You can delete unused code, we can always look back at older commits if we need to get this code back